easybuild/tools/filetools.py

```diff
@@ -768,7 +768,14 @@ def extract_cmd(filepath, overwrite=False):
         if 'unzip -qq' in cmd_tmpl:
             cmd_tmpl = cmd_tmpl.replace('unzip -qq', 'unzip -qq -o')
 
-    return cmd_tmpl % {'filepath': filepath, 'target': target}
+    if build_option('keep_current_timestamp'):
+         ts_tar='m'
```

Comment:
@jdonners Please use the long option `--touch` rather than `(-)m`, since it's an uncommon option to use with `tar`.

Reply:
Also: include spaces around `=`, so `ts_tar = '--touch '`

Reply:
well, the `--touch` option is specific to GNU tar, but wouldn't work for *BSDtar. So the long option would not work on OS/X, for example.

Reply:
Ugh, ok, then it indeed makes sense to stick to using `-m`.